### ISSUE TITLE
The link gives a 404 error when opened

### ISSUE THREAD

**Issue by HAWKM0TH** [The-Burning/blackeye-im#46]
![VirtualBox_Kali-Linux-2021 4a-virtualbox-amd64_27_01_2022_13_21_38](https://user-images.githubusercontent.com/98187202/151420626-f38e6d1e-d1a0-4e4a-aa23-e4a7e7674ade.png)
As you can see in the pictures when I select ngrok it says that commands are not found and doesn't give a shortened link. Plus when I open the longer link it shows this: 
<img width="1440" alt="Screen Shot 2022-01-27 at 1 22 58 PM" src="https://user-images.githubusercontent.com/98187202/151420649-b80f5d24-8800-4147-896d-39abad630a1f.png">
Since ngrok didn't work I selected Localtunnel to see if it would work.
![VirtualBox_Kali-Linux-2021 4a-virtualbox-amd64_27_01_2022_13_12_36](https://user-images.githubusercontent.com/98187202/151420847-70192a2b-f979-4c4d-ac59-623864531add.png)
It seems like it might work since both links are showing, but when I open one of the links, they just show a blank page with 404 in the top corner. 
<img width="1440" alt="Screen Shot 2022-01-27 at 1 12 19 PM" src="https://user-images.githubusercontent.com/98187202/151421017-7ba16cda-e15f-4290-b069-f224b1e43081.png">
If anyone could help, I'd really appreciate it!

**Comment by iAFIT** [comment]
the same problem any help?

**Comment by HAWKM0TH** [comment]
Not yet

**Comment by Nipa-Code** [comment]
I might know how to fix this error with Ngrok, if you haven't yet fixed it. If you still need help feel free to reply on me

**Comment by Nipa-Code** [comment]
@iAFIT , @HAWKM0TH 
Well I will explain it anyways: Open 2 terminals (one for blackeye that gives you the link) second one will be for the page you want to create the phishing page for.
For the second terminal do the following: 
<img width="604" alt="image" src="https://user-images.githubusercontent.com/60810623/153571471-39fbe5d8-4eb8-42b3-9b90-caef919688a3.png">
```
cd sites 
cd {site_name_you_want_to_create_the_phishing_page_for}
# **this is comment** : now you pick the website, I'm using netflix for the example but it doesn't matter Run the following
php -S localhost:8080
```
## A simple explanation of what happened above

So, what we do is that we start a php server on the website you want, just remember to pick right directory for it and you are all good to go. Now when you visit the link it should be working.

And here is the image from the first terminal (just showing what it looks like)
<img width="410" alt="image" src="https://user-images.githubusercontent.com/60810623/153572280-0d2ed15e-3e4e-4c9c-9a13-6431c7f8f44a.png">
## Recap
So quick recap: Open 2 terminals
Terminal 1. For blackeye
Terminal 2. For the site you want to make the phishing page for
On the Terminal 2. Type the following 
```
cd sites 
cd site_name_here 
php -S localhost:8080
````
We just started a php server for localhost. Now for the Terminal 1.
Start the blackeye using what ever command you are used to use eg. `./blackeye.sh`. then pick site and choice option 1, and vistit link and it should be fixed as long as the php server is running. Hope this helps you all!

**Comment by bjwarford08** [comment]
so i did everything still 404 but in the terminal it says development server started
help?

**Comment by bjwarford08** [comment]
> I might know how to fix this error with Ngrok, if you haven't yet fixed it. If you still need help feel free to reply on me

help

**Comment by DevMxnu** [comment]
Can you help me tnx

![Immagine 2022-10-22 151522](https://user-images.githubusercontent.com/86147961/197341067-c57e406c-128b-4054-8dc6-7eccd7f30092.png)

**Comment by Davidalfa081** [comment]
Yes I can help you but the developers get request for 15$ for the command
